fdbserver/storageserver.actor.cpp

```diff
@@ -3620,6 +3621,43 @@ bool storageServerTerminated(StorageServer& self, IKeyValueStore* persistentData
 		return false;
 }
 
+ACTOR Future<Void> memoryStoreRecover(IKeyValueStore* store, Reference<ClusterConnectionFile> connFile, UID id)
+{
+    if(store->getType() != KeyValueStoreType::MEMORY || connFile.getPtr() == nullptr) {
```

Comment:
This if-block has a different indentation then the rest of the function.

Reply:
run `git clang-format` on storageserver.cpp